src/dynapi/SDL_dynapi.h

```diff
@@ -43,7 +43,9 @@
 #include "TargetConditionals.h"
 #endif
 
-#if defined(TARGET_OS_IPHONE) && TARGET_OS_IPHONE  /* probably not useful on iOS. */
+#if !defined(SDL_LOADSO_DLOPEN) /* can't be used if the toolchain does not support dlfcn.h. */
```

Comment:
This is wrong: E.g. it excludes windows which otherwise supports SDL_DYNAMIC_API

Reply:
Fixed, thanks for spotting this mistake, I always forgot about other systems than Linux.